src/stories/molecules/GrafHeader/GrafHeader.tsx

```diff
@@ -13,27 +13,42 @@ import {
 } from 'stories/molecules/GrafHeader/GrafHeader.style';
 import { ReactComponent as Logo } from 'stories/assets/thriving_park_logo.svg';
 import SvgIcon from 'stories/atoms/SVGIcon/svgIcon';
+import { screenSizes, defaultScreenSize } from 'utils/constants';
 
 interface Props {
-  isMobile: boolean;
+  screenSize: string;
 }
 
-export const GrafHeader = ({ isMobile = false }: Props) => {
+export const GrafHeader = ({ screenSize = defaultScreenSize }: Props) => {
+  const getSize = () => {
```

Comment:
Since we can't use nested ternaries, a function is the only way I can think of if we are considering rendering 3 sizes based on 3 kinds of screen sizes. Please let me know if there are better ways.